### ISSUE TITLE
[Question] Finetune Performance Flustrated a lot!

### ISSUE THREAD

**Issue by WEIYanbin1999** [haotian-liu/LLaVA#961]
### Question

We are fine-tuning on our custom dataset of 850 <image, QA> pairs using the official finetune_task_lora.sh script. However, with identical parameters, hardware, environment, and seed, we observe wildly varying performance during inference, with accuracy ranging even 20%. We have tried modifying the deepseed strategy, running on multiple GPUs as well as a single GPU, and fixing the seed, but we are unable to achieve relative reproducibility in performance. We are seeking assistance in this matter.
![image](https://github.com/haotian-liu/LLaVA/assets/127707751/47a53057-5bc3-48f0-b69e-3d9100088699)

**Comment by yuezih** [comment]
+1 👀 I'm seeking assistance in this matter too

**Comment by yuezih** [comment]
Found a similar issue: https://github.com/haotian-liu/LLaVA/issues/864
